### PR TITLE
dynamic-page/cart

### PR DESCRIPTION
**Pull request by Etaizil**
This is the logic for the cart. There might be changes to other code files due to merges and conflicts solving, but it should be in accordance to the last changes in dev. I will work on the style next, but at least we will have a flow.

### CODE REVIEW

File: backend/views/Cart/Cart.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            <div class="product-quantity">
+              <button class="btn btn-dark minus-button" data-id="${cartItem.id}">-</button>
+              <p class="item-amount mt-2">${cartItem.quantity}</p>
+              <button class="btn btn-dark plus-button" data-id="${cartItem.id}">+</button>
+            </div>
+            <button class="btn btn-dark remove-item" data-id="${cartItem.id}">remove</button>
+          </div>
+        </div>
+      </div>
+      <hr class="mb-3">
```
Comment (roeezach) [inline backend/views/Cart/Cart.js:93]: this hr causing the screen to look weired when removing an item. try to remove several items from the cart and you will see some duplicated horizional lines

**Review by roeezach — CHANGES_REQUESTED** [review]
it looks really good and the mean functionality work.
we still have an enoying bug:
when removing one item from the cart and then we try to edit the quantity of the other it increased the value by two or decrease by two.
lets discusses this further and try to resolve it

File: backend/views/Cart/Cart.js (hunk 1 above)
Comment (Etaizil) [reply backend/views/Cart/Cart.js:93]: correct, i just moved it up into the div :) will push soon

Comment (Etaizil) [reply backend/views/Cart/Cart.js]: hr removed. product-container border instead.

**Review by roeezach — APPROVED** [review]

**Pull request merged**